Source/Domain/Iwentys.Domain/SubjectAssignments/Enums/AvailabilityState.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Iwentys.Domain.SubjectAssignments.Enums
+{
+    public enum AvailabilityState
+    {
+        Visible,
+        Hidden,
+        Deleted
```

Comment:
```suggestion
        Visible = 1,
        Hidden = 2,
        Deleted = 3
```

Чтобы потом не тратить 10500 часов на осознание, что умершая сериализация будет всегда возвращать Visible

Reply:
Валидно. Хорошая практика енамы с =1 начинать, чтобы отличать если случайно 0 прилетит.